bots/config.yaml

```diff
@@ -0,0 +1,26 @@
+organization: thoth-station
+repositories:
```

Comment:
What about querying for repos inside org?

Reply:
and blacklist the ones we dont want to be bot'erized?